### ISSUE TITLE
Bug: 0.9.91 causes key repeat issues

### ISSUE THREAD

**Issue by yingzhu146** [Hammerspoon/hammerspoon#3003]
Thanks a lot for this great lib! I have remapped tab/shift tab as follows to reduce RSI

```lua
-- alias
local function keyEvent(...)
    return hs.eventtap.event.newKeyEvent(...)
end
--alias
local function keyMap(...)
    return hs.keycodes.map[...]
end
--alias
local function hotkey(...)
    return hs.hotkey.bind(...)
end

local function shift_tab()
    return function()
        keyEvent(keyMap("shift"), true):post()
        keyEvent(keyMap("tab"), true):post()
        keyEvent(keyMap("tab"), false):post()
        keyEvent(keyMap("tab"), false):post()
        keyEvent(keyMap("shift"), false):post()
    end
end

local function tab()
    return function()
        keyEvent(keyMap("tab"), true):post()
        keyEvent(keyMap("tab"), false):post()
        keyEvent(keyMap("tab"), false):post()
    end
end

hotkey(cmd, "i", shift_tab() )
hotkey(cmd, "o",tab())


```

Now - using shift-tab/tab behaves odd
- after a non-deterministic number of uses, it will start to behave as cmd-tab. Sometimes it returns to normal by itself, sometimes I need to restart Hammerspoon
- I realize there's a million co-variates that might influence this/cause this and might be very hard to reproduce, but figured you might have a few good ideas where to debug/if there's good built-in tools for me to get to the bottom of this
- Interestingly enough, If I try to debug what keys are being registered using either `karabiner-eventviewer` or `keycodes`, whenever either of these two applications is in front, it will revert to normal behavior so I can't debug the actual registered keys

Apologies for the vague/irreproducible problem and thanks a ton for the help!

**Comment by yingzhu146** [comment]
Maybe relatedly but since the update to 0.9.91, other key bindings are no longer triggered reliably as well (started 1 week ago roughly, while the original problem started a few months ago so before update)

I have `right cmd + hjkl` mapped to the arrow keys, holding down command then keeping any of hjkl pressed will, very sporadically (but enough to be extremely irritating) not trigger correctly. this time, it does not get resolved switching to `keycodes` so here's a screenshot showing that indeed for some reason it doesn't realize that the modifier is down 
<img width="347" alt="image" src="https://user-images.githubusercontent.com/59233975/143284411-7c04a3e5-6215-413f-9239-46a00aae281d.png">

`20.6.0 Darwin Kernel Version 20.6.0: Mon Aug 30 06:12:20 PDT 2021; root:xnu-7195.141.6~3/RELEASE_ARM64_T8101 arm64` is my OS

**Comment by yingzhu146** [comment]
Appears solved by downgrading to 0.9.87.

**Comment by yingzhu146** [comment]
Also appears to be solved in newest version - closing
